commerce_coordinator/apps/rollout/waffle.py

```diff
@@ -23,3 +25,11 @@ def is_program_redirection_to_ct_enabled(request):
     Check if the PROGRAM_REDIRECT_TO_COMMERCETOOLS_CHECKOUT flag is enabled.
     """
     return waffle.flag_is_active(request, PROGRAM_REDIRECT_TO_COMMERCETOOLS_CHECKOUT)
+
+
+def is_order_fulfillment_service_redirection_enabled(request):
```

Comment:
```suggestion
def is_order_fulfillment_service_forwarding_enabled(request):
```

can we use the word `forwarding` instead of `redirection` since redirection is more of HTTP context word and its not truly a redirection either.